### ISSUE TITLE
`prefer-called-exactly-once-with` should report sole `toHaveBeenCalledTimes(1)`

### ISSUE THREAD

**Issue by FloEdelmann** [vitest-dev/eslint-plugin-vitest#869]
In this test case:

```js
        it('does something', async () => {
            const foo = bar()

            expect(baz).toHaveBeenCalledTimes(1)
        })
```

The test would be more exhaustive when using `toHaveBeenCalledExactlyOnceWith`

**Comment by nami8824** [comment]
I opened a related proposal for `prefer-called-with`:  https://github.com/vitest-dev/eslint-plugin-vitest/issues/910

**Comment by FloEdelmann** [comment]
@veritem #911 did not change the behavior reported in this issue: `toHaveBeenCalledTimes(1)` is still not reported.
